app/src/main/java/helium314/keyboard/latin/LatinIME.java

```diff
@@ -1592,6 +1602,12 @@ private void setSuggestedWords(final SuggestedWords suggestedWords) {
                 || noSuggestionsFromDictionaries) {
             mSuggestionStripView.setSuggestions(suggestedWords,
                     mRichImm.getCurrentSubtype().isRtlSubtype());
+            // Auto hide the toolbar if dictionary suggestions are available
+            if (currentSettingsValues.mAutoHideToolbar
+                    && !noSuggestionsFromDictionaries
+                    && !suggestedWords.isPrediction()) {
```

Comment:
Why don't hide if it's a prediction?

Reply:
I just wanted to know why you added it like this, it was not a criticism or request for change.

Now you changed it to `(!suggestedWords.isPrediction() || !currentSettingsValues.mAutoShowToolbar)) {`, but I'm still not sure I understand why. (and really, this is not a request to change it, just for clarification)